lesson2/2_4.js

```diff
@@ -0,0 +1,14 @@
+// Реализовать функцию для сортировки массива чисел по убыванию
+// const numbers = [-20, -10, 0, 10, 20, 30]
+// sortDesc(numbers) // [30, 20, 10, 0, -10, -20]
+
+const numbers = [-20, -10, 0, 10, 20, 30]
+
+const sortArray= function (a,b) { 
+  return b - a
+}
+function sortDesc(dataArray) {return dataArray.sort(sortArray)}
```

Comment:
Отлично, а теперь чтобы сократить код, можно переместить объявление функции сравнения в место её вызова с использованием синтаксиса стрелочных функций:

```suggestion
function sortDesc(dataArray) {return dataArray.sort((a, b) => b - a)}
```

Reply:
Или можно пойти еще дальше, и также записать вашу основную функцию как стрелочную:

```suggestion
const sortDesc = dataArray => dataArray.sort((a, b) => b - a)
```